IBM Power documentation for HCP {please table for post GA} (#5712)
open-cluster-management/rhacm-docs: clusters/hosted_control_planes/config_dns_ibmpower.adoc

[#configuring-dns-hosted-control-plane-ibmpower]

> = Configure DNS for hosted control plane with IBM Power

Reviewer: ```suggestion
= Configuring DNS for hosted control planes on IBM Power
```

> The API Server for the hosted cluster is exposed. A DNS entry must exist for the `api.<hosted_cluster_name>.<base-domain>` that points to the destination where the API server is reachable.

Reviewer: ```suggestion
The API server for the hosted cluster is exposed. A DNS entry must exist for the `api.<hosted_cluster_name>.<base-domain>` entry that points to the destination where the API server is reachable.
```

The DNS entry can be as simple as a record that points to one of the nodes in the managed cluster that is running the hosted control plane.

> The entry can also point to a load balancer that is deployed to redirect incoming traffic to the Ingress pods.

Reviewer: ```suggestion
The entry can also point to a load balancer that is deployed to redirect incoming traffic to the ingress pods.
```
The term "ingress" needs to be capitalized only in reference to the Ingress object.
